tutorials/plot_background_statistics.py

```diff
@@ -178,47 +179,45 @@ def plot_t_p(t, p, title, mcc, axes=None):
 # is true, we should be able to exchange conditions and not change the
 # distribution of the test statistic.
 #
-# In the case of a 2-tailed paired t-test against 0 (or between two conditions
-# where you have already subtracted them), exchangeability means that we can
-# flip the signs of our data. Therefore, we can construct the
-# **null distribution** values by taking random subsets of samples (subjects),
-# flipping the sign of their data, and recording the resulting statistic
-# absolute value. The absolute value of the statistic evaluated on the
-# veridical data can then be compared to this distribution, and the p-value
-# is simply the proportion of null distribution values that were smaller.
+# In the case of a two-tailed 1-sample t-test (which also tests the difference
+# between two conditions against zero), exchangeability means that we can flip
```

Comment:
I'd explicitly write that you could do a two-condition test via a 1-sampe test on their difference. Otherwise it's confusing because people might think of a 2-sample test when you talk of 2 conditions